### DISCUSSION TITLE
LOVE THIS SOLUTION !

### DISCUSSION THREAD

**Discussion by sshearer** [dialogic-godot/dialogic#637] [General]
Hi, I just wanted to offer my kudos for the current version in the app store.  I installed the current stable release of godot and your add-on yesterday, and it was my first time seeing either.  Within a few hours I had written my first non-interactive 'game' with background imagery, a few characters with default photo, text & associated audio dialog, and in 1 scene, flow from 1 timeline to the next.  VERY COOL.

But now I'm thinking about next steps: organizing & structuring a long series of resources and timelines to build an  educational video.  Ignoring the whole 'export to standard video formats' process, I have a couple questions:

1. The Dialogic view seems to imply only 1 instance, if you will.  Ie. I don't see how to create a new scene and drop in another Dialogic part, and see an empty set of characters, timelines, etc.  Actually,  I would want the ability to reuse all but the timelines in subsequent instances.  Am I correct that there's only support for 1 instance in the Project, or am I just uneducated?  If uneducated, is there a reference somewhere that shows how to do it ?  I've been through most of the videos and your documentation. 

2. IF multiple instances are supported, are there any guidelines on when it is best to define a new scene, and link timelines into it ?  I would image doing so where each godot scene relates to a scene in the video.  But maybe there are other considerations ?

**Comment by Jowan-Spooner** [comment]
Yeah, you guessed right, dialogic and everything in the Dialogic tab is global. The best solution for what you want is probably to create subfolders in the timeline folder for each thing you do.
